### PR TITLE
Fix the data load issue for structured files

### PR DESCRIPTION
**Pull request by XuhuiRen**
## Description

Fix the data loading issue for the structured data, e.g., excel, csv, and json/jsonl. Each single element in the structured file will be saved as a single instant in the knowledge base.

## Issues

n/a

## Type of change
Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: comps/dataprep/utils.py [hunk 1]
```diff
@@ -276,6 +276,7 @@ def load_json(json_path):
     """Load and process json file."""
     with open(json_path, "r") as file:
         data = json.load(file)
+    content_list = [json.dumps(item) for item in data]
```
Comment (XinyuYe-Intel) [inline comps/dataprep/utils.py:279]: add this line alone does change the function.

File: comps/dataprep/redis/langchain/prepare_doc_redis.py [hunk 2]
```diff
@@ -170,11 +170,19 @@ def ingest_data_to_redis(doc_path: DocPath):
 
     content = document_loader(path)
 
-    chunks = text_splitter.split_text(content)
+    structured_types = [".xlsx", ".csv", ".json", "jsonl"]
```
Comment (XinyuYe-Intel) [inline comps/dataprep/redis/langchain/prepare_doc_redis.py:173]: There are two more redis related code also need to change in this way.

Comment (XuhuiRen) [reply comps/dataprep/redis/langchain/prepare_doc_redis.py:173]: code for ray is fixed. code for llamaindex uses another data loader will not be effected

File: comps/dataprep/utils.py [hunk 3]
```diff
@@ -276,7 +276,8 @@ def load_json(json_path):
     """Load and process json file."""
     with open(json_path, "r") as file:
         data = json.load(file)
-    return json.dumps(data)
+    content_list = [json.dumps(item) for item in data]
```
Comment (lkk12014402) [inline comps/dataprep/utils.py:279]: what is the fields/format in json/jsonl file?
{"text": "xxx"}?
{"content": "xxx"}

I don't see how to extract the text from json/jsonl as chunk to ingest

File: comps/dataprep/qdrant/prepare_doc_qdrant.py [hunk 4]
```diff
@@ -47,11 +48,18 @@ def ingest_data_to_qdrant(doc_path: DocPath):
 
     content = document_loader(path)
 
-    chunks = text_splitter.split_text(content)
+    structured_types = [".xlsx", ".csv", ".json", "jsonl"]
```
Comment (lkk12014402) [inline comps/dataprep/qdrant/prepare_doc_qdrant.py:51]: the structured_types are used by many files, we should define it as common variable.

File: comps/dataprep/utils.py (hunk 3 above)
Comment (XuhuiRen) [reply comps/dataprep/utils.py:279]: considering that most of json/jsonl data exhibit their own data format in each line, e.g., {"text":xxx, "docid":xxx, "url":xxx}. We can directly output each line of data as a single data instance.

**Review by lkk12014402 — APPROVED** [review]

**Review by lvliang-intel — APPROVED** [review]

**Review by XinyuYe-Intel — APPROVED** [review]

**Pull request merged**